pkg/cmd/roachtest/test/test_interface.go

```diff
@@ -26,6 +28,24 @@ const DefaultCockroachPath = "./cockroach"
 // in that environment variable.
 var EnvAssertionsEnabledSeed = "ROACHTEST_ASSERTIONS_ENABLED_SEED"
 
+// EnvWorkloadDurationFlag - environment variable to override
+// default run time duration of workload set in tests
+// usage ROACHTEST_PERF_WORKLOAD_DURATION="5m"
+const EnvWorkloadDurationFlag = "ROACHTEST_PERF_WORKLOAD_DURATION"
+
+// ValidateWorkloadDurationFlag checks if EnvWorkloadDurationFlag
+// input is a valid duration string - matches the pattern
+func ValidateWorkloadDurationFlag(input string) bool {
```

Comment:
I agree with Renato's comment that it's kinda strange to have these definitions here. Let's maybe move them into `utils.go` (under `pkg/cmd/roachtest/tests`) until there is a more established mechanism for passing arbitrary key/val pairs into roachtests.